doc/source/dev/roadmap-detailed.rst

```diff
@@ -68,8 +68,7 @@ Benchmarks
 ``````````
 The ``asv``-based benchmark system is in reasonable shape.  It is quite easy to
 add new benchmarks, however running the benchmarks is not very intuitive.
-Making this easier is a priority.  In addition, we should run them in our CI
-(gh-8779 is an ongoing attempt at this).
+Making this easier is a priority.
```

Comment:
gh-8779 was merged, benchmarks run on CI now. 

Not sure if we still feel that running benchmarks is unintuitive; the process is not bad and it is documented [here](http://scipy.github.io/devdocs/dev/contributor/benchmarking.html#benchmarking-with-asv). 

We could consider removing that part and instead:
> The `asv`-based benchmark system is in reasonable shape, but we need a sustainable solution for tracking benchmark results on consistent hardware over time

@V0lantis, thought you might like to chime in here.

Reply:
`bench-compare` still fails fairly often. running with `--no-build` is the most reliable, but that doesn't work when comparing two commits.

Reply:
K we'll leave it as-is then.

Reply:
I will try to tackle this issue :-)